index.js

```diff
@@ -31,49 +38,50 @@ function Client(addr, opts) {
  */
 
 Client.prototype.call = function(method, params) {
-  var id = uid(16);
-
-  var body = {
+  const id = uid(16);
+  const body = {
     method: method,
     params: params,
-    id: id
+    id: id,
+    jsonrpc: '2.0' // http://www.jsonrpc.org/specification
   };
 
-  debug('call %s(%j) :: %s', method, params, id);
-
-  var opts = {
+  const opts = {
     json: true,
     method: 'POST',
     uri: this.addr,
     timeout: this.timeout,
     body: body
   };
 
-  return new Promise(function(resolve, reject){
-    request.post(opts, function(err, res, body){
-      body = body || {}
+  return new Promise(function(resolve, reject) {
+    debug('request %j', body);
+    request.post(opts, function(err, res, body) {
+      body = body || {};
 
       if (err) {
-        debug('error for %s: %s', id, err.message)
+        debug('error for %s: %s', id, err.message);
         return reject(err);
```

Comment:
yeah i don't mind either, i just want to be able to enable debug and see the response body.